config/roles.yml

```diff
@@ -262,6 +262,7 @@ ml_read_access:
     - 'cluster:admin/opensearch/ml/models/search'
     - 'cluster:admin/opensearch/ml/tasks/get'
     - 'cluster:admin/opensearch/ml/tasks/search'
+    - 'cluster:admin/opensearch/ml/model_groups/search'
```

Comment:
Could you add this action in alphabetic order please?

Reply:
Could you please suggest if this line should go at line 261 or 263?

Reply:
> @rbhavna Thanks - is there a specific version you'd like this change to be back ported into?

Yes, to 2.x and 2.8